quickstarts/nextcloud/config.yml

```diff
@@ -0,0 +1,39 @@
+slug: nextcloud
+
+description: |
+  ## Comprehensive monitoring quickstart for Nextcloud
+  With New Relic's Nextcloud monitoring system, keep an eye on your Nextcloud in real-time, correlate transactions moving across the application environment and troubleshoot the issue which you are facing.
```

Comment:
"You can use New Relic's infrastructure agent to monitor the performance of your Nextcloud server in near real-time, correlate transactions move across the app environment, and identify issues to troubleshoot."